Sources/Plasma/FeatureLib/pfPython/plPythonSDLModifier.cpp

```diff
@@ -322,37 +350,17 @@ bool plPythonSDLModifier::IPythonVarIdxToSDL(plSimpleStateVariable* var, int var
     case plVarDescriptor::kByte:
     case plVarDescriptor::kBool:
     case plVarDescriptor::kInt:
-        if (PyLong_Check(pyVar)) {
-            int v = PyLong_AsLong(pyVar);
-            var->Set(v, varIdx);
-            if (!hintstring.empty())
-                var->GetNotificationInfo().SetHintString(hintstring);
-            return true;
-        } else if (PyLong_Check(pyVar)) {
```

Comment:
:trollface:

Reply:
Presumably one of those used to be a `PyInt_*`...  I hope >.>